applications/openshift/scheduler/scheduler_profiling_protected_by_rbac/rule.yml

```diff
@@ -0,0 +1,93 @@
+documentation_complete: true
+prodtype: ocp4
+title: |-
+  Verify that the scheduler API service is protected by RBAC
```

Comment:
@rhmdnd I just noticed that the title and description match `1.4.2`. I'm confused, :joy:

Reply:
Ah - fixing!